cliquet/resource/__init__.py

```diff
@@ -1046,6 +1088,17 @@ def get_parent_id(self, request):
         """
         return ''
 
+    def _extract_fields(self):
+        """Override default fields extraction from QueryString to allow
+        the permission_field to exist.
+        """
+
+        fields = super(ShareableResource, self)._extract_fields()
+        if fields:
+            fields.append(self.model.permissions_field)
+
+        return fields
```

Comment:
I think this is more than partial responses. I will open a new issue.

I will remove the code, then asking for partial response will hide permissions. I spotted a "bug" where asking for collection endpoint adds __permission__. This was not intended. Correct behavior should be clearly defined.